public/store/results/actions.ts

```diff
@@ -584,6 +584,7 @@ export const actions = {
       solution,
       key,
       label,
+      "summary",
```

Comment:
Maybe add some string constants for "summary" and "confidence" but it's not a blocker.

Reply:
Added some constants to cover those two values